rero_ils/modules/circ_policies/api.py

```diff
@@ -265,6 +264,7 @@ def allow_request(cls, item, **kwargs):
         library_pid = kwargs['library'].pid if kwargs.get('library') \
             else item.library_pid
         cipo = cls.provide_circ_policy(
+            item.organisation_pid,
```

Comment:
to be tested : Is the `replace_refs()` call (behind the `organisation_pid` property) has better performance than a single DB call if the object has multiple references inside.